homeassistant/components/fyta/image.py

```diff
@@ -0,0 +1,63 @@
+"""Entity for Fyta plant image."""
+
+from __future__ import annotations
+
+from datetime import datetime
+
+from homeassistant.components.image import ImageEntity, ImageEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import FytaConfigEntry
+from .coordinator import FytaCoordinator
+from .entity import FytaPlantEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: FytaConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the FYTA plant images."""
+    coordinator = entry.runtime_data
+
+    description = ImageEntityDescription(key="plant_image", name="picture")
```

Comment:
why is the name `picture`. In this case we might consider just setting the name to `None`. We do that with entities that are the main feature of the device, well and in this case we don't really have much main features, but then it will just be `image.my_plant` instead of `image.my_plant_picture`. As in, it's an image. Of course its a picture.